src/shared/data/communication.data.ts

```diff
@@ -0,0 +1,22 @@
+export const RS_DOCS_COMMUNICATION_LINK = 'https://docs.rs.school/#/rs-school-chats';
+
+export const discordLinks = {
+  javascript: 'https://discord.com/invite/QvEYg7EaQ4',
+  'javascript-en': 'https://discord.com/invite/uW5cCHR',
+  react: 'https://discord.com/invite/zyRcphs3px',
+  'react ru': 'https://discord.com/invite/zyRcphs3px',
+  angular: 'https://discord.com/invite/xwReXYqvs7',
+  'node.js': 'https://discord.com/invite/8BFb8va',
+  'aws fundamentals': 'https://discord.com/invite/WEZxwRa4J6',
+  'aws cloud dev': 'https://discord.com/invite/WEZxwRa4J6',
+};
+
+export type CourseNames =
```

Comment:
Do we really need this type?

Reply:
I found more elegant and generic way, in my opinion: 
```TS
export type CourseNames = keyof typeof DISCORD_LINKS;
```